JS/js-forms/js-forms_checkbox-input/js/index.js

```
@@ -11,13 +12,29 @@ function showTosError() {
   tosError.removeAttribute("hidden");
 }
 
+function hideSuccess() {
+  success.setAttribute("hidden", "");
+}
+
+function showSuccess() {
+  success.removeAttribute("hidden");
+}
+
+hideTosError();
+hideSuccess();
```

Why are we hiding these things using JS? This means they will be shown briefly until this piece of code is run. If they're supposed to be hidden by default, it makes more sense to hide them directly in the markup (html).

---

I see. Then either: 1. I'm missing some context (haven't done this challenge yet) as to why this is needed or 2. It might be worth asking whoever told you to do this why this is necessary :)